docs/tool/js/views/map-view.js

```diff
@@ -932,6 +931,32 @@
                         value = (value === null | value == "null") ? ' Unknown' : value; // handles when data has "null" as a value
                         dl.append('dd').text(value)
                     }
+
+                    var currentNlihc = projectData.properties.nlihc_id;
+                    var subsidyContainerId = 'subsFor' + currentNlihc;
+                    var subsidyTableId = 'subsTableFor' + currentNlihc;
+                    controller.getData({
+                        name: currentNlihc + '_subsidy',
+                        url: "http://hiapidemo.us-east-1.elasticbeanstalk.com/api/project/" + currentNlihc + "/subsidies",
+                        callback: function(data){
+                            current.append('div')
+                                .attr('id', subsidyContainerId)
+                                .append('p')
+                                .text('Subsidies for this project');
+                            d3.select('#' + subsidyContainerId)
+                                .append('div').attr('id', subsidyTableId);
+                                                
+                            new D3Table('#' + subsidyTableId)
+                                .data(data.items)
+                                .columns([
+                                    {field:'poa_end', label:'Scheduled End Date', class:'value', html: function(d){return d}},
```

Comment:
Let's change these labels to be shorter given the space constraints. How about just "Start" and "End". I also think we should maybe only display the start/end year, to save space - users can see the full date by clicking through to the proejct page. Finally, do put a "-" or similar null character marker if there is missing data like date.